apps/server/src/shared/repo/task/task.repo.integration.spec.ts

```diff
@@ -985,7 +985,45 @@ describe('TaskRepo', () => {
 			});
 		});
 
-		describe('when user is the creator', () => {
+		describe('when parent is "not" passed', () => {
+			it('should "not" find finished tasks', async () => {
+				const user = userFactory.build();
+				const course = courseFactory.build({ students: [] });
+				const task = taskFactory.finished(user).build({ course });
+
+				await em.persistAndFlush([task]);
+				em.clear();
```

Comment:
Any need why em is cleared here but not in the next case?

Reply:
no particular reason. the em.clear() makes sure the orm cache doesnt mess with any results.
I added it to the new tests as well.